proxy/hdrs/MIME.h

```diff
@@ -1856,6 +1857,8 @@ MIMEHdr::set_age(time_t value)
     if (sizeof(time_t) > 4) {
       value_set_int64(MIME_FIELD_AGE, MIME_LEN_AGE, value);
     } else {
```

Comment:
Maybe just get rid of the 32 check.  I don't think we support any OSes that have a 32bit time_t.